### ISSUE TITLE
Panic

### ISSUE THREAD

**Issue by eskimo** [a16z/helios#153]
Finally got it running only to now get this output:

```
thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', /home/eth/helios/execution/src/evm.rs:283:26
thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', /home/eth/helios/execution/src/evm.rs:283:26
thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', /home/eth/helios/execution/src/evm.rs:283:26
thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', /home/eth/helios/execution/src/evm.rs:283:26
thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', /home/eth/helios/execution/src/evm.rs:283:26
thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', /home/eth/helios/execution/src/evm.rs:283:26
thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', /home/eth/helios/execution/src/evm.rs:283:26
```

**Comment by ncitron** [comment]
Do you happen to know what call your rpc is making when it crashes? This looks like it may be a bug which we've seen happen intermittently, so we'd appreciate knowing all the ways to trigger it.

We're tracking that issue at #133

cc @refcell

**Comment by eskimo** [comment]
> Do you happen to know what call your rpc is making when it crashes? This looks like it may be a bug which we've seen happen intermittently, so we'd appreciate knowing all the ways to trigger it.
> 
> We're tracking that issue at #133
> 
> cc @refcell

Unfortunately I do not. These calls are being made by https://github.com/lukeburns/hipr-eth if that helps?

**Comment by refcell** [comment]
Fixed in #208
